src/entities/activity/lib/services/AnswersUploadService.ts

```diff
@@ -320,6 +321,13 @@ class AnswersUploadService implements IAnswersUploadService {
 
     const modifiedBody = await this.uploadAllMediaFiles(body);
 
+    const updatedUserActions = this.mapUserActionsMediaFilesToDto(
```

Comment:
Plase think how to try-catch this function, maybe add an internal one and wrap it with try-catch.

Reply:
Show me via teams, please when you will have a solution